scss/components/_tooltips.scss

```diff
@@ -0,0 +1,77 @@
+.oo-tooltip::before {
+  content: attr(data-tip) ;
+  font-family: "Roboto Condensed", sans-serif;
+  text-shadow: none;
+  position:absolute;
+  z-index: 99999999;
+  background: black;
+  color: white;
+  border-radius: 3px;
+  opacity: 0;
+  font-size: 15px;
+  padding: 8px 20px 8px 20px;
+  bottom: 59px;
```

Comment:
Is it possible to make this a function of `$control-bar-height` so that the offset doesn't break if we change the style of the control bar? A fixed value will also make it harder for customers who have custom skin styles.